pkg/runner/runner.go

```
@@ -187,7 +187,7 @@
 	return nil
 }
 
-func newCertStore() *certStore {
+func newCertStore(unused string) *certStore {
```

golangci-lint (1.62.2)

[warning] unused-parameter: parameter 'unused' seems to be unused, consider removing or renaming it as _

(revive)